hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -118,20 +134,41 @@ public static void main(String[] args) {
     jsc.stop();
   }
 
+  private static void validateRunningMode(Config cfg) {
+    // --mode has a higher priority than --schedule
+    // If we remove --schedule option in the future we need to change runningMode default value to EXECUTE
+    if (StringUtils.isNullOrEmpty(cfg.runningMode)) {
```

Comment:
Can we also add a validation here that if the mode is execute, then instant-time should not be empty and it should be a valid instant time?

Reply:
done